js/src/lib/components/InferenceWidget/shared/WidgetHeader/WidgetHeader.svelte

```diff
@@ -8,20 +8,27 @@
 	import ModelPipelineTag from "../../../ModelPipelineTag/ModelPipelineTag.svelte";
 
 	export let noTitle = false;
```

Comment:
@gary149  we still need it `noTitle` prop since it is used a few times in moon-landing like [here](https://github.com/huggingface/moon-landing/blob/89abbf5836da97c61a03639ff340afe9a223b576/server/views/pages/HomePage/HomePage.svelte#L284)